package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@eeacms/volto-eea-kitkat",
-  "version": "10.5.0",
+  "version": "10.5.1",
```

Comment:
@nileshgulia1 Maybe we should bump this to 11.0.0

Reply:
hm indeed. 🤔